plugins/node/opentelemetry-instrumentation-restify/test/restify.test.ts

```diff
@@ -109,7 +109,7 @@ const createServer = async (setupRoutes?: Function) => {
 
     server.use(useHandler);
     server.get('/route/:param', getHandler);
-    server.get('/thowing', throwError);
+    server.get('/throwing', throwError);
```

Comment:
I guess the test isn't actually using this test endpoint. :)

Reply:
Apparently not 🙃